L1Trigger/L1TMuonOverlap/python/simOmtfDigis_cfi.py

```diff
@@ -14,7 +14,8 @@
                               eventsXMLFiles = cms.vstring("TestEvents.xml"),
                               dropRPCPrimitives = cms.bool(False),                                    
                               dropDTPrimitives = cms.bool(False),                                    
-                              dropCSCPrimitives = cms.bool(False)   
+                              dropCSCPrimitives = cms.bool(False),
+                              #ghostBusterType = cms.string("GhostBusterPreferRefDt")
```

Comment:
so is this GhostBuster effectiveley never used so far, and left there for future studies? (How) was it tested?

Reply:
@rekovic could you please clarify the point above? Apart for than that I am ok with integrating this PR by this evening

Reply:
Thanks @fabiocos That is correct. 
The code "as is" was tested with unit tests showing no differences in emulated muons.  "Ghost-busting" when turned on was tested by the developers.